Workflow/Sources/RenderContext.swift

```diff
@@ -148,4 +148,11 @@ extension RenderContext {
                 }
             }
     }
+
+    /// Generates a sink that allows sending the Workflow's output wrapped in an AnyWorkflowAction, allowing bypassing an
+    /// intermediate action.
+    public func makeOutputSink(of output: WorkflowType.Output) -> Sink<WorkflowType.Output> {
```

Comment:
should just be `func makeOutputSink() -> Sink<WorkflowType.Output>` since we want to be able to send the output to the resulting sink, not at creation time (as evidenced by `output` never being used here)

Reply:
Oh you're totally right. 

Surprised xcode didn't alert me that the params were not used...